README.md

```diff
@@ -106,6 +106,41 @@ cd Scripts
 pipenv run python helmet.py
 ```
 
+
+### Mock assignment run in Helmet UI
+
+Download [Helmet-UI executable](https://github.com/HSLdevcom/helmet-ui/releases/). 
+
+#### Change the config settings
+1. Download latest version of helmet-model-system in `Projektin asetukset` (edit: or set this manually to `helmet-model-system\Scripts` folder which you probably downloaded when installing Python environment!).
+2. Find out where that version is saved by hovering over the text box (for example: `C:\Users\xxx\helmet-model-system-v4.1.3-24-04-1684924655775`)
+3. Open `dev-config.json` inside that previous folder and modify OPTIONAL_FLAGS argument:
+    -     `"OPTIONAL_FLAGS": ["DO_NOT_USE_EMME"]`
+	
+#### Set Helmet UI settings
+1. Click "Projektin asetukset" button
+    - Python from previous virtual environment, for example `C:\Users\xxx\.virtualenvs\Scripts-AJXyDTuP\Scripts\python.exe`
+    - helmet-model-system was set previously
+    - Projektin kansiopolku: your choice
+    - Lähtödata: `helmet-model-system\tests\test_data\Base_input_data`
+    - Tulokset: your choice (copy Matrices folder from to the location) or `helmet-model-system\tests\test_data\Base_input_data`
+2. "Uusi Helmet-skenaario" button
+    - Skenaarion nimi: your choice
+    - Emme-projekti: create an empty (dummy) text file and save it with file extension `.emp`
+    - Liikenneverkon sisältävä skenaario: default
+    - Syöttötiedot: `helmet-model-system\tests\test_data\Scenario_input_data\2030_test`
+    - Iteraatioiden enimmäismäärä: default (do not cross "loppusijoittelu")
+    - The rest can have default values (edited) 
+	
+#### Run your first demo model run
+1. Copy `helmet-model-system\tests\test_data\Results\test\Matrices` as your `[result-folder]\[scenario-name]\Matrices` folder.
+2. Select scenario with checkbox and click `Käynnistä (1) skenaariota`.When running the first model run again and again, just do #2. When you create more scenarios, always copy matrices (#1) first to a new location. (This is a demo model thing that needs to be done because we are independent of Emme.)
+	
```

Comment:
We could remove this part and instruct the user to set result folder to `helmet-model-system\tests\test_data\Results` and scenario name as `test`. And say that if the user wants to test more scenarios, they can create a copy of `Results\test` (because matrices need to be copied).